packages/story-editor/src/app/currentUser/currentUserProvider.tsx

```diff
@@ -49,13 +57,18 @@ function CurrentUserProvider({ children }) {
   }, [currentUser, getCurrentUser]);
 
   const updateCurrentUser = useCallback(
-    (data) =>
-      _updateCurrentUser ? _updateCurrentUser(data).then(setCurrentUser) : null,
+    (data: updateCurrentUserProps) =>
+      _updateCurrentUser
+        ? void _updateCurrentUser(data).then(setCurrentUser)
```

Comment:
Need to remove the `void` here because other parts of the code base expect `updateCurrentUser` to return a Promise that they can use `.catch()` on in case of errors

Reply:
Catch needs to do done elsewere.